raiden_libs/test/mocks/client.py

```diff
@@ -207,9 +207,9 @@ def close_channel(self, partner_address: Address, balance_proof: BalanceProof):
     def settle_channel(
         self,
         partner_address: Address,
-        transferred=tuple(),
-        locked=tuple(),
-        locksroot=tuple()
+        transferred=tuple(),  # noqa: B008
+        locked=tuple(),  # noqa: B008
+        locksroot=tuple()  # noqa: B008
```

Comment:
trailing comma here

Reply:
the build actually has a lot of warnings for trailing commas, was it decided to not fail if there are flake8 violations? (Is this consistent across our repos?)

Reply:
Asked the same in rocket chat. That's why I mentioned the script :)

Reply:
But @hackaugusto if you check Travis it did fail. The `make lint` command returned `2`. It's just that we got multiple commands to run so all of them run ([as per the docs](https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Build-Step)) and all errors are accumulated.

If we want to avoid that (which would be a good idea to save travis times) we should do the following:

```diff
script:
-  - make lint
- - make typecheck
- - make test
+ make lint && make typecheck && make test
```

Reply:
ah, I see, I had the impression travis stopped execution on the first error